styles/style.css

```diff
@@ -202,13 +263,18 @@ p {
   text-decoration: underline;
 }
 
+.article__content{
+  }
+
 .fa {
   margin-right: 0.5rem;
   color: var(--orange-dark);
 }
 
 .facebook-link {
   color: var(--orange-dark);
+  /* Right-Justify the Logo*/
+  margin-left:12rem;
```

Comment:
I wouldn't use a specific value to right-justify an element. Screen sizes may vary and it may not be 12 rem to right-justify the FB link. Since this is a text, you can use `text-align` on the parent div, or if you still want to use the flexbox, then try playing around with the `justify-content` property